panel/models/echarts.py

```diff
@@ -54,6 +54,8 @@ def __js_skip__(cls):
 
```

Comment:
The new parameter should also be added to the Reference Notebook documentation. We need the parameter described in the parameter section. I think it could be helpful if the parameter description contained "... for example `options_kwargs={ "replaceMerge": ['series'] })`".

Maybe a link to the apache echart `setOption` options would be helpful?

Reply:
Done in my latest commit, with a link to the documentation of setOption :)